src/qtism/common/beans/Bean.php

```diff
@@ -228,24 +249,27 @@ public function getSetter($property)
 
     /**
      * Whether the bean has a valid setter for a property with name $propertyName.
-     * A setter is considered to be valid if:
+     * A setter is considered to be valid if:.
```

Comment:
after this statement follow list of cases when setter considered as valid so i guess point not applicable here